src/NuGet.Core/NuGet.Packaging/Signing/Utility/VerificationUtility.cs

```diff
@@ -160,7 +159,7 @@ internal static SignatureVerificationStatusFlags ValidateTimestamp(Timestamp tim
                         validationFlags |= SignatureVerificationStatusFlags.IntegrityCheckFailed;
                     }
                 }
-                catch
+                catch (ArgumentException)
```

Comment:
We don't want to swallow all the exceptions here, just the ones expected (i.e. one of these methods can throw a `PlatformNotSupportedException`)